wasm/jit/jit_amd64.go

```diff
@@ -175,23 +175,6 @@ func (c *amd64Compiler) compileHostFunction(address wasm.FunctionAddress) error
 	if err := c.callGoFunction(jitCallStatusCodeCallHostFunction, address); err != nil {
 		return err
 	}
-
```

Comment:
PR comment why we aren't looking at this anymore?

Reply:
this was accidentally added so I just deleted!